App.tsx

```diff
@@ -2,18 +2,32 @@ import React from 'react';
 import {NavigationContainer} from '@react-navigation/native';
 import {RootNavigator} from './src/navigation/RootNavigator';
 import RNBootsplash from 'react-native-bootsplash';
+import {store, RootStoreContext} from './src/models';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {persist} from 'mst-persist';
+
+const RootStore = store.create({
+  count: 0,
+});
 
 function App(): JSX.Element {
+  persist('RootStore', RootStore, {
```

Comment:
We should just call a method called setupStore from within model and it should take care of all this

Reply:
Prajna Can you share me something to read about this setupStore, cause i can't find it and it is mentioned in the documentation and the example as well to follow this pattern.

https://github.com/agilgur5/react-native-manga-reader-app/blob/master/App.js#L11

Reply:
As discussed, the code is fine. Just wanted you to move it to model folder as its a code related to model. If we start adding initialisation codes of everything in App.tsx then the file length will be huge and hard to read